fase_di_sviluppo/src/modules/homework.js

```diff
@@ -5,22 +5,145 @@
  * defines a school task.
  * 
  * @file This file defines Homework class.
- * @author Andrea Lanzani
+ * @author Riccardo Ronconi
  * Everyone who modifies this file MUST add his/her name
  * @version 1.0alpha
  */
 
+import { Duration } from './duration.js';
+import { Subject } from './subject.js';
+
 /**
  * @todo Implement this class.
  * @todo Write the documentation.
  * 
  * Class representing a school homework.
  */
 export class Homework {
+
   /**
    * Create a homework.
+   * 
+   * @param {string} title - The title of the homework.
+   * @param {Date} deadline - The date of the homework's deadline.
```

Comment:
La descrizione ha poco senso in inglese
```suggestion
   * @param {Date} deadline - The homework's deadline.
```